src/main/java/redis/clients/jedis/EntryID.java

```diff
@@ -0,0 +1,103 @@
+package redis.clients.jedis;
+
+import java.io.IOException;
+import java.io.Serializable;
+
+public class EntryID implements Comparable<EntryID>, Serializable{
```

Comment:
How about `StreamEntryID` instead of `EntryID`?

Reply:
Since there's no real association to a specific Stream and the same EntryID can be used with couple of Streams I thought it's better to keep it as EntryID